uwuno/src/main/java/com/learning/uwuno/util/utils.java

```diff
@@ -6,5 +6,18 @@ private utils() {
         // Should not run
     }
 
+    // Returns a string containing all characters before the last symbol given
```

Comment:
Going to leave this here for now, this was previously being used but I changed implementation. But maybe these will be useful in the future?

Reply:
Sounds good.